src/components/Home/Home.scss

```diff
@@ -1,3 +1,122 @@
+.clearfix:after {
+  content: "";
+  display: table;
+  clear: both;
+}
+
 .homeWrapper {
   width: 100%;
+}
+
+.topTriangle {
+	position: absolute;
```

Comment:
seems like maybe you have tabs turned on in Sublime? please switch to two spaces